README.md

```diff
@@ -0,0 +1,75 @@
+#zendev
+
+##Description
+zendev takes care of setting up and managing the disparate components of a standard Zenoss development environment, 
+with or without the control plane. It's primary usecase is to checkou out all github repositories needed for 
```

Comment:
"with or without the control plane" - maybe "with or without Control Center"?

also, "chekou out" should be "checkout"